charts/vald/templates/index/job/readreplica/rotate/configmap.yaml

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2019-2023 vdaas.org vald team <vald@vdaas.org>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+{{- $rotator    := .Values.manager.index.readreplica.rotator -}}
+{{- $gateway    := .Values.gateway.lb -}}
+{{- $index      := .Values.manager.index -}}
+{{- $agent      := .Values.agent -}}
+{{- $discoverer := .Values.discoverer -}}
+{{- if $rotator.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $rotator.name }}-config
+  labels:
+    app.kubernetes.io/name: {{ include "vald.name" . }}
+    helm.sh/chart: {{ include "vald.chart" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Chart.Version }}
+    app.kubernetes.io/component: {{ $rotator.name }}
+data:
+  config.yaml: |
+    ---
+    version: {{ $rotator.version }}
+    time_zone: {{ default .Values.defaults.time_zone $rotator.time_zone }}
+    logging:
+      {{- $logging := dict "Values" $rotator.logging "default" .Values.defaults.logging }}
+      {{- include "vald.logging" $logging | nindent 6 }}
+    server_config:
+      {{- $servers := dict "Values" $rotator.server_config "default" .Values.defaults.server_config }}
+      {{- include "vald.servers" $servers | nindent 6 }}
+    observability:
+      {{- $observability := dict "Values" $rotator.observability "default" .Values.defaults.observability }}
+      {{- include "vald.observability" $observability | nindent 6 }}
+    rotator:
+      agent_namespace: {{ $rotator.agent_namespace | quote }}
+      read_replica_label_key: "vald-readreplica-id"
+      read_replica_id: "_MY_TARGET_REPLICA_ID_"
+      volume_name: "vald-agent-ngt-readreplica-pvc"
```

Comment:
Is it better to set the agent name based on values.yaml instead of the const variable? 🤔 
https://github.com/vdaas/vald/blob/3ec66bf0cee67fda3c96a52ec2a0ef08caaab021/charts/vald/values.yaml#L1431

Reply:
These parameters will be replaced with `values.yaml` settings of readreplica deployment in the next PR.